lambda/translcore.ml

```diff
@@ -719,10 +719,7 @@ and transl_apply ~scopes
            if we already passed here this is a no-op. *)
         let l =
           List.map
-            (fun (arg, opt) ->
-               match arg with
-               | Omitted () -> arg, opt
-               | Arg arg -> Arg (protect "arg" arg), opt)
+            (fun (arg, opt) -> Typedtree.map_apply_arg (protect "arg") arg, opt)
```

Comment:
(I think this change and the one below were meant to go in the *previous* commit, but it's no big deal and fine to leave as-is.)

Reply:
I decided to defy Flemme and made the change :-)